src/i18n/en/index.json

```diff
@@ -1,2 +1,28 @@
 {
+  "results-per-page": "Results per page",
```

Comment:
To start with proper base, perhaps use snake case for simpler alignment moving forward?

Reply:
done